src/common/app.config.js

```diff
@@ -106,6 +106,7 @@ const appConfig = /* @ngInject */ function (envServiceProvider, $compileProvider
     SEND_MESSAGE_TO: 'Send a Message to {{ministry}}',
     SPECIAL_INSTRUCTIONS: 'Special Handling Instructions for Processing This Gift',
     MESSAGE_EXAMPLE: 'For example: make this gift anonymous, stop my gift after 18 months, etc.',
+    COMMENTS_DISABLED: 'Comments are disabled when Special Handling Instructions are being sent.',
```

Comment:
Should probably put this on both translations (even though we only have English right now).